oval/types.go

```diff
@@ -89,16 +89,26 @@ type Reference struct {
 // Advisory : >definitions>definition>metadata>advisory
 // RedHat and Ubuntu OVAL
 type Advisory struct {
-	XMLName         xml.Name   `xml:"advisory"`
-	Severity        string     `xml:"severity"`
-	Cves            []Cve      `xml:"cve"`
-	Bugzillas       []Bugzilla `xml:"bugzilla"`
-	AffectedCPEList []string   `xml:"affected_cpe_list>cpe"`
-	Refs            []Ref      `xml:"ref"` // Ubuntu Only
-	Bugs            []Bug      `xml:"bug"` // Ubuntu Only
-	PublicDate      Date       `xml:"public_date"`
-	Issued          Date       `xml:"issued"`
-	Updated         Date       `xml:"updated"`
+	XMLName         xml.Name    `xml:"advisory"`
+	Severity        string      `xml:"severity"`
+	Cves            []Cve       `xml:"cve"`
+	Bugzillas       []Bugzilla  `xml:"bugzilla"`
+	AffectedCPEList []string    `xml:"affected_cpe_list>cpe"`
+	Refs            []Ref       `xml:"ref"` // Ubuntu Only
+	Bugs            []Bug       `xml:"bug"` // Ubuntu Only
+	PublicDate      Date        `xml:"public_date"`
+	Issued          Date        `xml:"issued"`
+	Updated         Date        `xml:"updated"`
+	Affected        []Affected2 `xml:"affected"`
```

Comment:
This is probably Red Hat, only, right? Let's check, and if that's the case, let's add a comment similar to the comments on other fields `// Red Hat Only`